Milestone 1. Created website for Dasik Events

indigodavid opened the pull request:
During the current milestone I created a website with two pages, following the template by Cindy Shin. The theme of the site is about the band I have with my brother, Dasik. The site has the following features:

- Home page and About page
- Responsive design with breakpoint at 768px
- Javascript programmed menu for mobile version
- Clipping background letters for home page and animation
- Dynamically generated content for speakers section

indigodavid committed:
Add index.html and style.css

indigodavid committed:
Add about.html and general html structure overall

indigodavid committed:
Add CSS for mobile menu and intro section

indigodavid committed:
Edit section program

indigodavid committed:
Edit mobile versions of index.html and about.html

indigodavid committed:
Add background clipping effect to letters

indigodavid committed:
Edit events section on about.htmlfor desktop

indigodavid committed:
Add menu behavior in Javascript

indigodavid committed:
Modify html with custom data

indigodavid committed:
Add pattern for speakers photos

indigodavid committed:
Add Javascript to generate dynamically Speakers section

indigodavid committed:
Fix linters errors

Kingstalux commented on js/speakers.js:
```diff
@@ -0,0 +1,56 @@
+  },
+];
+
+function generateLi(index) {
+  const li = document.createElement('li');
+  const photo = document.createElement('div');
+  const info = document.createElement('div');
+  const h3 = document.createElement('h3');
+  const h4 = document.createElement('h4');
+  const p = document.createElement('p');
+
+  photo.classList.add('photo');
+  info.classList.add('info');
+
+  photo.innerHTML = `<div class="pattern"> </div><img src="${speakerData[index].image}" alt="${speakerData[index].name}"></div>`;
+  h3.innerText = speakerData[index].name;
+  h4.innerText = speakerData[index].role;
+  p.innerHTML = speakerData[index].description;
+
+  info.appendChild(h3);
+  info.appendChild(h4);
+  info.appendChild(p);
+
+  li.appendChild(photo);
+  li.appendChild(info);
+  return li;
```
- [x] Great job generating your speaker's section dynamically from JavaScript 💯 In the mobile view, only 2 speakers should be displayed together with a `more` button. Kindly rectify this issue.
- [x] (OPTIONAL) Also when a user clicks the `more` button the rest of the speakers are then displayed, it would be great if you could implement this functionality so as to improve the UI of your site 👍 
- [x] Kindly add the red horizontal line under `band collaborators` so as to stick to the original design.
- Your design:
![Annotation 2022-05-19 040711](https://user-images.githubusercontent.com/85113567/169196729-c41b8eda-edfb-462b-94f0-39125f1e38d5.jpg)
- Required design:
![Annotation 2022-05-19 040653](https://user-images.githubusercontent.com/85113567/169196764-62ca0d93-03e5-477c-bb49-da6ff99744db.jpg)

Kingstalux commented on about.html:
```diff
@@ -39,36 +39,39 @@
-            <h3>2015</h3>
-            <p>Lorem ipsum dolor sit amet.</p>
+            <h3>2022</h3>
+            <p>"La Aldea" Pub Gig</p>
           </div>
         </li>
         <li>
-          <img src="./images/event.jpg" alt="Event">
+          <img src="./images/AfterGig2.jpeg" alt="After">
           <div class="overlay">
-            <h3>2013</h3>
-            <p>Lorem ipsum dolor sit amet.</p>
+            <h3>2021</h3>
+            <p>"After" Pub Gig</p>
           </div>
```
- [x] Great design and animations in this section 💯 It would be better to lay the red overlay with the text as default so as to follow the original design, then when a user hovers over the image, the red overlay and text then disappear.
- Your design:
![Annotation 2022-05-19 040902](https://user-images.githubusercontent.com/85113567/169197164-beedc65b-68fe-4c6c-8e40-3d0eb759e5fd.jpg)

- Required design:
![Annotation 2022-05-19 040840](https://user-images.githubusercontent.com/85113567/169197219-8eaf9dd5-991f-4432-a90f-fb708b2c3995.jpg)

Kingstalux commented on index.html:
```diff
@@ -25,62 +25,62 @@
-          <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Nemo, vitae?</p>
+          <i class="fa-solid fa-video"></i>
+          <h3>Video</h3>
+          <p>La Aldea Pub Concert<br>Katari Gig</p>
```
- Good job in this program section but it still needs a few finishing touches 😄 
- [x] First, your program section is missing a button underneath it in the mobile version and an underlined text in the desktop version. Kindly add those.
- [x] Also, it would be great if you could align the texts as in the original design. Right now your text alignment is out of sync.
- [x] Next, in the desktop view, you could add the white border hover effect as shown in the original design. This will improve the UI of your site.
- Your desktop design:
![Annotation 2022-05-19 040501](https://user-images.githubusercontent.com/85113567/169197786-af9943b8-077d-4689-b7cf-26c69ba8cde8.jpg)
- Required desktop design:
![Annotation 2022-05-19 040447](https://user-images.githubusercontent.com/85113567/169197824-a759e5d1-d5d5-4686-b4e6-15aaa6420578.jpg)

- Your mobile design:
![Annotation 2022-05-19 040605](https://user-images.githubusercontent.com/85113567/169197859-8e7af0e1-d886-4602-ae4f-9d40f997a8d6.jpg)
- Required mobile design:
![Annotation 2022-05-19 040621](https://user-images.githubusercontent.com/85113567/169197880-3e8581bb-50e7-4a32-9e47-4affdea4dd21.jpg)

Kingstalux commented on about.html:
```diff
@@ -77,7 +77,7 @@ <h3>2013</h3>
+        <img src="./images/cc-logo-white.png" alt="Creative Commons">
       </div>
       <div class="cp-info">
         <p><a rel="license" href="http://creativecommons.org/licenses/by-nc/4.0/"><img alt="Creative Commons License" style="border-width:0" src="https://i.creativecommons.org/l/by-nc/4.0/88x31.png" /></a></p>
```
- [x] You'll notice that even though the desktop version of the about page has a dark footer, the mobile version instead has a dark partner's section and a white footer. Kindly address this issue so as to follow the design requirements 👍  
- Your design:

![Annotation 2022-05-19 040746](https://user-images.githubusercontent.com/85113567/169198252-b13d3d10-d545-4691-9631-1679fcbb831a.jpg)
- Required design:
![Annotation 2022-05-19 040803](https://user-images.githubusercontent.com/85113567/169198316-d32d93ef-3102-46bc-aa09-13435371cba5.jpg)

Kingstalux reviewed (changes requested):
Hi @indigodavid, 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!
![68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f6e59597451644c7177776c54653363634b682f67697068792e676966](https://user-images.githubusercontent.com/85113567/169198395-026d2a8d-acd8-42d4-90f1-e05af4b1b5fc.gif)

## Highlights 👏 
✅ Great video presentation 👍 
✅ Good commit history 💯 
✅ Good responsive design 👏 
✅ Great use of animations and transitions 🥂 


## Suggested changes
_Check the comments under the review._

- [x] It would be great if you could add the link to your video presentation to your README file. It would make it more professional 👍 

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

indigodavid committed:
Modify template requirements

- Add video link to README
- Add More button with JS implementation to hide and show speakers in
mobile version
- Add red bar to speakers
- Invert the red overlay behavior
- Add program section link with its mobile formatting
- Modify text alignment for the program section
- Add the white border hover effect for the program section children
- Modify the footer behavior for the mobile section

indigodavid committed:
Fix linters errors

danmainah commented on about.html:
```diff
@@ -0,0 +1,95 @@
+      </ul>
+    </div>
+  </section>
+  <footer id="copyright" class="cp-dark">
+    <div class="wrapper">
+      <div class="footer-logo">
+        <img id="img-white" src="./images/cc-logo-white.png" alt="Creative Commons">
+        <img id="img-dark" src="./images/cc-logo-dark.png" alt="Creative Commons">
+      </div>
+      <div class="cp-info">
+        <p><a rel="license" href="http://creativecommons.org/licenses/by-nc/4.0/"><img alt="Creative Commons License" src="https://i.creativecommons.org/l/by-nc/4.0/88x31.png" /></a></p>
+        <p>2022 Creative Commons. Some Rights Reserved</p>
+        <p>This work is licensed under a <a rel="license" href="http://creativecommons.org/licenses/by-nc/4.0/">Creative Commons Attribution-NonCommercial 4.0 International License</a>.</p>
+      </div>
+    </div>
+  </footer>
```
- [ ] [OPTIONAL] Kindly remove the error code displayed below your footer, see the image below
![image](https://user-images.githubusercontent.com/61408860/169390351-42a3ebbb-6ce3-47ae-b47e-f7cd8dcb0b2c.png)

danmainah reviewed (approved):
Hi @indigodavid, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good design
✔️  Good PR and Description

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
